packages/core/labels.scss

```diff
@@ -0,0 +1,40 @@
+@import "./variables/all";
+@import "./mixins/all";
+@import "./functions";
+
+.jkl-form-support-label {
+    display: block;
+    margin-top: $component-spacing--small;
+    @include font-size(xs);
+    @include line-height(xs);
+
+    &--help {
+        color: $svart;
+    }
+
+    &--error {
+        color: $error-color;
+    }
+}
+
+.jkl-label {
+    display: inline-block;
+    @include font-size("xl");
```

Comment:
Since font-size and line height normally goes together in the same size, should that be default behavior, and we can override in other cases?

Reply:
Not a bad idea. For this application we should probably implement an analogue to `@font-size/@line-height` for the spacing as well.